l10n_cl_financial_indicators/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Chilean Financial Indicators",
```

Comment:
This seems rather specifically related to currencies, and not to generically financial /accounting related, correct?
If so I suggest to improve and narrow down the modules name and description.

Reply:
Daniel, what would be the best approach to the name? (in Spanish this is referenced as "indicadores financieros" that's why the name was that).

A proposal could be "l10n_cl_special_currency_rates" for example. What do you think?

Reply:
Even that, UF and UTM are not currencies; they are indexes or indicators but should be treated inside Odoo as currencies.

Reply:
I will close this, since can see that @nelsonramirezs already change the module name.

Thnks!